crates/bevy_pbr/src/render/mesh_view_types.wgsl

```diff
@@ -85,3 +85,8 @@ struct ClusterOffsetsAndCounts {
     data: array<vec4<u32>>,
 };
 #endif
+
+struct Globals {
+    time: vec4<f32>,
+    dt: vec4<f32>,
+}
```

Comment:
Missing newline. :)

Reply:
And do structs not need a ; after the closing} anymore?

Reply:
Well, I didn't get any errors for not having it. Not sure if that's normal.

Reply:
no more final ; is required on current wgsl draft, makes me very happy!!

other note, could `dt` be named more descriptively? `delta` or `delta_time` seems a lot more clear to me, would maintain consistency with other bevy code as well.

Reply:
> no more final ; is required on current wgsl draft, makes me very happy!!

That's good to know

> other note, could dt be named more descriptively? delta or delta_time seems a lot more clear to me, would maintain consistency with other bevy code as well.

Yeah, good call, I threw this together pretty quickly. I'll go with delta_time